improver/metadata/forecast_times.py

```diff
@@ -190,6 +190,7 @@ def _create_frt_type_coord(cube, point, name="forecast_reference_time"):
     new_points = round_close([coord_units.date2num(point)], dtype=coord_type_spec.dtype)
     new_coord = cube.coord(frt_coord_name).copy(points=new_points)
     new_coord.rename(name)
+    new_coord.attributes = {}
```

Comment:
This rather opaque statement to remove any attributes that might be present could be avoided, along with the rename, by simply creating the coord like this (sadly, `"blend_time"` isn't a standard_name, which `rename` handles much better):
```suggestion
    try:
        new_coord = DimCoord(new_points, standard_name=name, units=coord_units)
    except ValueError:
        new_coord = DimCoord(new_points, long_name=name, units=coord_units)
```

Reply:
OK